packages/lib/src/toolbar/controls/Selector/Selector.tsx

```diff
@@ -143,31 +142,29 @@ function Selector<T extends string>(props: Props<T>) {
                 </Option>
               ))
             ) : (
-              <ul className={styles.list}>
+              <ul className={styles.groups}>
                 {Object.entries(options).map(([groupLabel, groupOptions]) => (
                   <li key={groupLabel}>
                     <span className={styles.groupLabel}>{groupLabel}</span>
-                    <ul className={styles.list}>
```

Comment:
Removing the list markup inside each group. Too much noise for screenreader users.